integration-tests/__tests__/before-each-queue.js

```diff
@@ -13,6 +13,6 @@ const runJest = require('../runJest');
 describe('Correct beforeEach order', () => {
   it('ensures the correct order for beforeEach', () => {
     const result = runJest('before-each-queue');
-    expect(result.stdout).toMatchSnapshot();
+    expect(result.stdout.replace(/\\/g, '/')).toMatchSnapshot();
```

Comment:
or use `slash`, it's in the deps

Reply:
😅 oops too late

Reply:
Just checked the `slash` code, and it doesn't seem to be used for non-path strings: https://github.com/sindresorhus/slash/blob/master/index.js. It does a non-ASCII search before replacing, which means further changes adding one of these characters would make the test fail.